Docker/dbadmin.Dockerfile

```diff
@@ -3,26 +3,21 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-#edfialliance/ods-api-db-admin:7.1
-FROM edfialliance/ods-api-db-admin:7.1@sha256:36e4295bf125eb804fbde1381d95361e8723b5ff2bcac173854fc2897e811c79 as base
-LABEL maintainer="Ed-Fi Alliance, LLC and Contributors <techsupport@ed-fi.org>"
+FROM edfialliance/ods-api-db-admin:7.2.5570-5.1.0@sha256:53a9c8bbed76075bc438e63ba9579a48f1539bf7d5a3ee1b5171739c7e496ab0 AS base
+USER root
+RUN apk --upgrade --no-cache add dos2unix=~7 unzip=~6
 
-ENV POSTGRES_USER=${POSTGRES_USER}
-ENV POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-ENV POSTGRES_DB=postgres
```

Comment:
These three environment variables are setup using a `.env` file at runtime. There is no reason to create the variables here in the Dockerfile, and there might even be a security problem if you do so.